core-js/src/main/javascript/cdf/components/ToggleButtonBaseComponent.js

```diff
@@ -70,7 +70,7 @@ define(['../lib/jquery', './InputBaseComponent'], function($, InputBaseComponent
           }
         }
 
-        if(myself.type == 'radio' || myself.type == 'radioComponent') {
+        if(myself.type == 'radio' || myself.type == 'RadioComponent') {
```

Comment:
You will also need to update the CDF (AMD) sample for RadioComponent:
[doc](https://github.com/webdetails/cdf/blob/master/assemblies/cdf-samples/src/main/resources/plugin-samples/pentaho-cdf/pentaho-cdf-require/30-documentation/30-component_reference/10-core/52-RadioComponent/template.html#L21)
[sample code](https://github.com/webdetails/cdf/blob/master/assemblies/cdf-samples/src/main/resources/plugin-samples/pentaho-cdf/pentaho-cdf-require/30-documentation/30-component_reference/10-core/52-RadioComponent/template.html#L122)
It should also be <code>type: "RadioComponent"</code> to match what the CDE (AMD) renderer outputs.
And maybe you can even fix the **MultiButtonComponent** sample:
[doc](https://github.com/webdetails/cdf/blob/master/assemblies/cdf-samples/src/main/resources/plugin-samples/pentaho-cdf/pentaho-cdf-require/30-documentation/30-component_reference/10-core/56-MultiButtonComponent/template.html#L21)
[sample code](https://github.com/webdetails/cdf/blob/master/assemblies/cdf-samples/src/main/resources/plugin-samples/pentaho-cdf/pentaho-cdf-require/30-documentation/30-component_reference/10-core/56-MultiButtonComponent/template.html#L126)
It should be <code>type: "MultiButtonComponent"</code> to match what the CDE renderer outputs.